README.md

```diff
@@ -27,6 +27,13 @@ such as Unity packages and images. Please install the latest git-lfs before clon
 which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions). 
 To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
+Recommend to run these samples on HoloLens 2 using following version of tools:
```

Comment:
Do we want to add an MRTK version recommendation too?

Reply:
Fixed in latest iteration.